### PR TITLE
Allow split coverage strings to be properly translated

### PR DESCRIPTION
**Pull request by Faalagorn**
#### Summary
None

#### Purpose of change
Allow some of the missing coverage strings to be translated, as they weren't if the string was full.

#### Describe the solution
Split the strings into two halves.

#### Describe alternatives you've considered
Mark just the " The " string to translate, it will require separate translations but won't result in split <info> and </info>, they will be hardcoded however. 

#### Testing
Compiled the game, with the strings already translated properly (removed "The" for Polish), it appears correctly.

### CODE REVIEW

**Comment by Faalagorn** [comment]
I mentioned it in Discord, but when I tried the alternative approach I got an error (type mismatch?) when compiling, but since my programming skills are extremely dull, I'm not sure how would it be solved.

I'm not 100% sure which approach is better anyway, but just in case here's the commit for my 2nd attempt in my local fork https://github.com/Faalagorn/Cataclysm-DDA/commit/0fe6ffc5c2b1a06bdf0a782e08de245d94c79e73

File: src/item.cpp [hunk 1]
```diff
@@ -3531,7 +3531,7 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
                 temp = sbp->opposite;
             } else {
                 // if it doesn't have an opposite print it alone and continue
-                coverage += _( " The <info>" + sbp->name + "</info>" );
+                coverage += _( " The <info>" ) + sbp->name +  _( "</info>" );
```
Comment (BrettDong) [inline src/item.cpp:3534]: I prefer `string_format( _( “The <info> %s </info>” ), sbp->name )`

File: src/item.cpp [hunk 2]
```diff
@@ -3543,7 +3543,7 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
                 // go through each body part and test if its partner is there as well
                 if( temp == *sbp_it ) {
                     // add the multiple name not the single
-                    coverage += _( " The <info>" + sbp->name_multiple + "</info>" );
+                    coverage += string_format( _( "The <info> %s </info>" ), sbp->name_multiple );
```
Comment (BrettDong) [inline src/item.cpp:3546]:
```suggestion
                    coverage += string_format( _( " The <info>%s</info>" ), sbp->name_multiple );
```

File: src/item.cpp [hunk 3]
```diff
@@ -3555,7 +3555,7 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
             }
             // if we didn't find its pair print it normally
             if( !found ) {
-                coverage += _( " The <info>" + sbp->name + "</info>" );
+                coverage += string_format( _( "The <info> %s </info>" ), sbp->name );
```
Comment (BrettDong) [inline src/item.cpp:3558]:
```suggestion
                coverage += string_format( _( " The <info>%s</info>" ), sbp->name );
```

**Review by BrettDong — COMMENTED** [review]
In current version there is a space before "The" and there is no space between <info> tag and body part name.

File: src/item.cpp [hunk 4]
```diff
@@ -3531,7 +3531,7 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
                 temp = sbp->opposite;
             } else {
                 // if it doesn't have an opposite print it alone and continue
-                coverage += _( " The <info>" + sbp->name + "</info>" );
+                coverage += string_format( _( "The <info> %s </info>" ), sbp->name );
```
Comment (wapcaplet) [inline src/item.cpp:3534]:
```suggestion
                coverage += string_format( _( " The <info>%s</info>" ), sbp->name );
```

I think the leading space was there to separate the addition from whatever may already be in the `coverage` variable. There were also no spaces inside the tag before.

File: src/item.cpp (hunk 2 above)
Comment (wapcaplet) [inline src/item.cpp:3546]:
```suggestion
                    coverage += string_format( _( " The <info>%s</info>" ), sbp->name_multiple );
```

File: src/item.cpp (hunk 3 above)
Comment (wapcaplet) [inline src/item.cpp:3558]:
```suggestion
                coverage += string_format( _( " The <info>%s</info>" ), sbp->name );
```

**Comment by wapcaplet** [comment]
Sorry to butt in, looks like @BrettDong and I just submitted the same suggestions. I'll retract mine.

**Pull request merged**